src/server/connection.rs

```diff
@@ -3114,6 +3142,42 @@ impl Connection {
         self.inner.send(msg.into());
         self.supported_encoding_flag = (true, not_use);
     }
+
+    #[cfg(not(any(target_os = "android", target_os = "ios")))]
+    async fn handle_window_focus(&mut self) {
+        if let Ok(displays) = super::display_service::update_get_sync_displays().await {
+            let current_display = crate::get_focused_display(displays);
+            if let Some(d_index) = current_display {
+                let mut misc = Misc::new();
+                misc.set_follow_current_display(d_index as i32);
+                let mut msg_out = Message::new();
+                msg_out.set_misc(misc);
+                self.send(msg_out).await;
+            }
+        }
+    }
+
+    #[cfg(not(any(target_os = "android", target_os = "ios")))]
+    async fn handle_cursor_switch_display(&mut self, pos: CursorPosition) {
```

Comment:
1. When there are multiple displays, this is still running, but the current display is wrong, and messages will to be sent repeatly.
2. use `get_sync_displays`, both for `handle_window_focus`